.github/workflows/code.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         shell: bash -l {0}
         run: |
           set -vxeuo pipefail
-          pip install -e .
+          pip install -e ".[dev]"
```

Comment:
The quotes are not necessary for the bash shell used in this workflow.  Please remove.

Reply:
done.